lms/static/js/edxnotes/views/shim.js

```diff
@@ -58,6 +58,31 @@ define([
         return (timeToExpiry > 0) ? timeToExpiry : 0;
     };
 
+    Annotator.Plugin.Tags.prototype.pluginInit = _.compose(
+        function() {
+            // Add screen reader label for edit mode.
+            var index = this.annotator.editor.fields.length -1;
+            var tagsField = 'annotator-field-' + index;
+            $('<label class="sr" for='+ tagsField +'>' + _t('Edit tags') + '</label>').insertBefore(
+                $('#'+tagsField, this.wrapper)
+            );
+            return this;
+        },
+        Annotator.Plugin.Tags.prototype.pluginInit
+    );
+
+    Annotator.Plugin.Tags.prototype.updateViewer = _.compose(
+        function() {
+            // Add aria information for viewing mode.
```

Comment:
Nit: the acronym is ARIA so I'd rather see it written that way in our code too.

http://a11yproject.com/posts/getting-started-aria/